pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -170,6 +170,7 @@
         if (conf.getTopicNames().isEmpty()) {
             setState(State.Ready);
             subscribeFuture().complete(MultiTopicsConsumerImpl.this);
+            conf.getTopicNames().add(singleTopic);
```

Comment:
@xuesongxs Could you give more details about how this change will fix the #9953? The singleTopic in the MultiTopicsConsumerImpl.java will always be a dummy topic name, in this case, you only add the dummy topic name into the topic config.

Reply:
@codelipenghui , The topic in the topicNames is deleted. After the consumer is created, the topic is not added to the topicNames, and the conf object is not passed to the MultiTopicsConsumerImpl constructor. Therefore, the original configuration of the consumer cannot be obtained in the MultiTopicsConsumerImpl constructor, and only a single input topic can call the MultiTopicsConsumerimpl < T > function Create partitioned consumer interface. At this time, singleTopic is the input topic. If there are multiple input topics, singleTopic is dummy topic.